Node-DC-EIS-client/node_dc_eis_testurls.py

```diff
@@ -191,16 +202,31 @@ def post_url(url,request_num,log,phase):
   global http_err
   global bad_url
   global post_datalist
+  global static_post
+  header_len = 0
+  content_len = 0
   r = None
   post_data = static_postdata
   if post_datalist:
     post_data = post_datalist[0]
+  else:
+    static_post = static_post + 1
   start = time.time()
   end = start
   for i in range(100):
     r =  post_function(url, post_data)
     if r:
       end = time.time()
+      content_len = len(r.content)
```

Comment:
please extract the length calculation into a separate function and call it from the post and get